hap.sh

```diff
@@ -116,11 +116,29 @@ do
 		for(i=1;i<=length(bm);i++) printf " %d has %d;",bm[i],numMatches[bm[i]];
 		print ""
 		PrintGroups(H);
+
+        ambigCount++
+        SetCopy(ambigReads[ambigCount], L)
 	    }
 	}
 	END{
 	    print "FINAL GROUPS"
-	    PrintGroups(H);
+        PrintGroups(H);
+        print ""
+        print "AMBIGUOUS READS"
+
+        PROCINFO["sorted_in"]="@ind_num_asc";
+
+        count = 0
+        for (a in ambigReads) {
+            count++
+            printf "Ambiguous[%d]:", count
+            for (r in ambigReads[a]) {
```

Comment:
Again so far as I can see this will just print out every read that came from every line, and it'll also print out any given read multiple times--once per line it appears. (Although now that I think of it, it's perhaps not the READS that are ambiguous but the lines that don't get added to a group.)